### PR TITLE
Replacing XPS broken links with suggested new page

### LINKED ISSUE #10528: Broken links
https://www.microsoft.com/xps

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/wpf/advanced/how-to-programmatically-print-xps-files.md#L60

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/wpf/advanced/printing-and-print-system-management.md#L24

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/wpf/advanced/printing-how-to-topics.md#L49

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/wpf/advanced/printing-overview.md#L26

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/wpf/advanced/printing-overview.md#L141

### PR DESCRIPTION
**Pull request by nemrism**
Change XPS web site broken links with the new page proposed by @NextTurn 

Fixes #10528

### CODE REVIEW

**Review by nxtn — CHANGES_REQUESTED** [review]
Thanks for replacing the links. I've left a small comment for you to consider.

**Review by nxtn — APPROVED** [review]
LGTM

File: docs/framework/wpf/advanced/printing-overview.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ With Microsoft .NET Framework, application developers using Windows Presentation
...
+ [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)] is an electronic document format, a spool file format and a page description language. It is an open document format that uses [!INCLUDE[TLA#tla_xml](../../../../includes/tlasharptla-xml-md.md)], [!INCLUDE[TLA#tla_opc](../../../../includes/tlasharptla-opc-md.md)], and other industry standards to create cross-platform documents. [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)] simplifies the process by which digital documents are created, shared, printed, viewed, and archived. For additional information on [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)], see the [XPS Documents](/windows/desktop/printdocs/documents).
```
Comment (rpetrusha) [inline docs/framework/wpf/advanced/printing-overview.md]:
```suggestion
 [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)] is an electronic document format, a spool file format and a page description language. It is an open document format that uses [!INCLUDE[TLA#tla_xml](../../../../includes/tlasharptla-xml-md.md)], [!INCLUDE[TLA#tla_opc](../../../../includes/tlasharptla-opc-md.md)], and other industry standards to create cross-platform documents. [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)] simplifies the process by which digital documents are created, shared, printed, viewed, and archived. For additional information on [!INCLUDE[TLA2#tla_metro](../../../../includes/tla2sharptla-metro-md.md)], see [XPS Documents](/windows/desktop/printdocs/documents).  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for addressing #10528, @nemrism. I've left one suggestion. After you incorporate it, we'll merge.

**Review by rpetrusha — APPROVED** [review]
Thanks for addressing #10528 with this PR, @nemrism. (And thanks for your review, @NextTurn.) I'll merge your PR now. The changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**